.github/dependabot.yml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: cargo
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 25
+  milestone: 2
+  reviewers:
+    - "mobilecoinfoundation/fogcore"
```

Let's have dependabot check the Actions, too.
```suggestion
    - "mobilecoinfoundation/fogcore"
- package-ecosystem: github-actions
  directory: "/"
  schedule:
    interval: daily
  open-pull-requests-limit: 10
  reviewers:
    - "mobilecoinfoundation/fogcore"
```